dev.zed.Zed.yaml

```diff
@@ -0,0 +1,101 @@
+# dev.zed.Zed.yaml
+app-id: dev.zed.Zed
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: Zed
+separate-locales: false
+finish-args:
+  # Grant access to Direct Rendering Infrastructure (DRI) for GPU acceleration
+  - --device=dri
+
+  # Set library paths for dynamically linked libraries
+  - --env=LD_LIBRARY_PATH=/app/lib:/usr/lib:/usr/lib/x86_64-linux-gnu/GL/default/lib
+
+  # Set configuration directories for XDG base directory specification
+  - --env=XDG_CONFIG_DIRS=/etc/xdg:/usr/lib/x86_64-linux-gnu/GL
+
+  # Set data directories for XDG base directory specification
+  - --env=XDG_DATA_DIRS=/app/share:/usr/lib/extensions/vulkan/share:/usr/share:/usr/share/runtime/share:/run/host/user-share:/run/host/share
+
+  # Allow access to the user's home directory for reading and writing files
+  - --filesystem=home
+
+  # Allow shared memory access for inter-process communication
+  - --share=ipc
+
+  # Enable network access
+  - --share=network
+
+  # Provide access to X11 display server as a fallback
+  - --socket=fallback-x11
+
+  # Enable access to PulseAudio sound server
+  - --socket=pulseaudio
+
+  # Enable access to Wayland display server
+  - --socket=wayland
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+  - org.freedesktop.Sdk.Extension.rust-stable
+
+inherit-extensions:
+  - org.freedesktop.Platform.GL.default:
+      directory: lib/GL
+      version: '23.08'
+      subdirectories: true
+      no-autodownload: false
+      autodelete: false
+      add-ld-path: lib
+      merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d
+      download-if: active-gl-driver
+      enable-if: active-gl-driver
+      autoprune-unless: active-gl-driver
+
+modules:
+  - name: zed
+    buildsystem: simple
+    build-commands:
+      - |
+        # Install scripts
+        install -Dm 755 apply_extra --target-directory ${FLATPAK_DEST}/bin/
+        install -Dm 755 zed --target-directory ${FLATPAK_DEST}/bin/
+
+        # Append environment variables for build flags
+        export CFLAGS+=' -ffat-lto-objects'
+        export CXXFLAGS+=' -ffat-lto-objects'
+
+        # Enable rust
+        source /usr/lib/sdk/rust-stable/enable.sh
+
+        # Build Zed with parallel jobs
+        cargo build --release --locked --all-features --jobs $(nproc)
```

Comment:
MUST use `--offline`

Reply:
[I will refactor the manifest to be offline](https://github.com/flathub/flathub/pull/5253#issuecomment-2118603243)